### PR TITLE
First cut at a bibliography, drawn from footnotes.

### PR DESCRIPTION
**Pull request by rwatson**

### CODE REVIEW

File: src/bibliography/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+[*Arm Architecture Reference Manual Supplement - Morello for A-profile
+Architecture*](https://developer.arm.com/documentation/ddi0606/latest),
+Document Number DDI0606 Version A.k, Arm Limited, 2022.
+
+Thomas Bauereiss, Brian Campbell, Thomas Sewell, Alasdair Armstrong,
+Richard Grisenthwaite, Graeme Barnes, Robert N. M. Watson, Simon W. Moore,
+Peter Sewell, and Jonathan Woodruff.
+[*The Arm Morello Evaluation Platform—Validating CHERI-Based Security in a
+High-Performance
+System*](https://developer.arm.com/documentation/ddi0606/latest) , IEEE Micro,
```
Comment (jrtc27) [inline src/bibliography/README.md]:
```suggestion
System*](https://developer.arm.com/documentation/ddi0606/latest), IEEE Micro,
```

**Review by jrtc27 — COMMENTED** [review]
Should we be doing a more standard [1] etc links to here for what are currently footnotes rather than duplicating things without proper linking/referencing beyond the names being the same?

**Comment by rwatson** [comment]
> Should we be doing a more standard [1] etc links to here for what are currently footnotes rather than duplicating things without proper linking/referencing beyond the names being the same?

I’ve updated the format a bit so that we still use footnotes, but only to hold the target title with link, plus a link to the bibliography. Possibly better? I still want the link on the page itself for web purposes.

**Review by jrtc27 — APPROVED** [review]
That works, just please squash the two bibliography commits before merging

**Pull request merged**